src/driver/mod.rs

```diff
@@ -10,6 +12,9 @@ cfg_if::cfg_if! {
     } else if #[cfg(target_os = "linux")] {
         mod iour;
         pub use iour::*;
+    } else if #[cfg(all(not(target_os = "linux"), unix))]{
```

Comment:
```suggestion
    } else if #[cfg(unix)]{
```

`else` means not linux.